apps/utils/teams_migration.py

```diff
@@ -0,0 +1,15 @@
+def _get_default_team(apps):
+    Team = apps.get_model("teams", "Team")
+    return Team.objects.order_by("id").first()
+
+
+def assign_model_to_team_migration(model_name, delete_if_no_team=False):
+    def _migration_op(apps, schema_editor):
+        model_cls = apps.get_model(model_name)
+        default_team = _get_default_team(apps)
+        if default_team:
+            model_cls.objects.update(team=default_team)
+        elif delete_if_no_team:
```

Comment:
I'm struggling to understand when/why this is needed?

Reply:
During initial setup (and during tests) there is are no teams so we delete the global default and then when new teams are created the team default get's created.

Reply:
Got it. So the code should not enter this branch in prod then